### DISCUSSION TITLE
INPLEMENTAZIONE AREA GESTIONE CORSI FORMATVI - PON

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#328] [Idee e proposte]
Buonasera ,sarebbe possibile integrare nel registro un'area relativa alla gestione dei corsi formativi ,pon etc... in con i vari attestati di completamento dei corsi in modo da contare anche i crediti scolastici partendo proprio da lì? Grazie 

Se magari conoscete altre piattaforme open source o comunque gratuite me le potete  scrivere qui sotto ?Hrazie

**Comment by trinko** [comment]
No, queste funzionalità al momento non sono previste. Non posso aiutarti.
